rdbms/types/table.hpp

```diff
@@ -22,6 +22,7 @@ class table
         void add_row(tuples rowTuple);
         void remove_row(tuples rowTuple);
         std::set<tuples, tuplesComparator> get_rows();
+         int get_rows_size(table T);
```

Comment:
why do you need this method?

Reply:
To get the size of the row for future scalability of the tuples.cpp